initial_x: 0.5
initial_y: 0.5
initial_z: 0.5
initial_roll: 0.0
initial_pitch: 0.0
initial_yaw: 0.0
from: world
to: thing
file_package: tf_keyboard_cal
file_name: /config/tf_keyboard_world_to_thing.yaml
topic_name: /keyboard_world_to_thing/keydown
